recipe/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+if [[ `uname` == 'Darwin' ]]; then
+    export MACOSX_DEPLOYMENT_TARGET=10.9
```

Comment:
I think it is possible to compile with `10.7` now that we have the `toolchain` but at the time that I was working there was no clear guideline.